### PR TITLE
Add initial 2019 chapter survey analysis

### PR DESCRIPTION
**Pull request by lorenanicole**
# Overview

This notebook starts digging into the feedback for the 2019 chapter organizer survey and to provide analysis on it #25.

In the notebook the responses (found in the PyLadies shared organizer folder for those with @pyladies.com email addresses) are normalized to help us identify the number of unique chapter responses and that data is joined with world cities data to determine overall number of countries responding.

## Future work 

Work to be done after this:

- Analyze text of things that need help
- Demonstrate how many chapters active within last 3 months, 6 months, 1 year

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.DS_Store
 .idea
-.csv
+*.csv
```
Comment (pfctdayelise) [inline .gitignore]: I think adding .ipynb_checkpoints/* (and removing the file from the repo) may be prudent.

Comment (lorenanicole) [reply .gitignore]: Oops clearly didn’t `git rm —cached` ha thanks for checking. Will update when home.

Comment (lorenanicole) [reply .gitignore]: Oops clearly didn’t `git rm —cached` ha thanks for checking. Will update when home.

Comment (lorenanicole) [reply .gitignore]: updated!

**Comment by pfctdayelise** [comment]
LGTM as a start. I don't know where the shared organizer folder is (it doesn't magically show up in my pyladies google drive) but I could possibly try and help with further analysis!

**Pull request merged**